backends/p4tools/common/lib/format_int.h

```diff
@@ -10,6 +10,14 @@
 
 namespace P4Tools {
 
+/// Defines common formatting options.
+struct FormatOptions {
+    bool useSep = false;
+    bool pad = true;
+    bool usePrefix = true;
+    bool writeUpperCaseHex = true;
```

Comment:
Nit: any reason why we say "write" here, but "use" for some of the other options? If not, shall we be consistent?